### DISCUSSION TITLE
Ask Siri for remaining time

### DISCUSSION THREAD

**Discussion by humpel111** [QuickSander/homebridge-mieleathome#48] [Q&A]
Hi QuickSander,

thanks a lot for the plug-in. I just implemented it to my hoobs and my washer dryer is exposed to home. While I can see the time remaining in home, I can not manage to ask the right question to make Siri tell the time remaining. Is there any question I can ask Siri to tell the information?

Thanks in advance,
Peter

**Comment by KingJayR1** [comment]
I have the same question xD

Did you find out? 

No matter what I ask Siri it tends to just turn of the washing machine instead of telling the remaining time

Thanks,

John
